fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/AvailabilityTestingUtils.groovy

```diff
@@ -62,4 +62,18 @@ class AvailabilityTestingUtils extends Specification {
                     table.setAvailability(new ConcreteAvailability(table.getTableName(), table.getSchema().getColumns(), new TestDataSourceMetadataService(allIntervals)))
                 }
     }
+
+    /**
+     * Construct a map of Column object to its all available intervals
+     *
+     * @param map a map of Column in String representation to its all available intervals in String representations
+     *
+     * @return a map of Column object to its all available intervals
+     */
+    static Map<Column, Set<Interval>> constructColumnToIntervalMap(Map<String, Set<String>> columnToIntervalInString) {
+        return columnToIntervalInString.collectEntries{
+            key,
+            value -> [ (new Column(key)): value.collect{new Interval(it)} ]
```

Comment:
Actually, I don't think this method is even used. Perhaps we can just drop it entirely? (IntelliJ doesn't think so, anyways)